ext/filter/filter_private.h

```diff
@@ -55,13 +55,27 @@
 #define FILTER_FLAG_NO_RES_RANGE            0x400000
 #define FILTER_FLAG_NO_PRIV_RANGE           0x800000
 
-#define FILTER_FLAG_HOSTNAME               0x100000
+#define FILTER_FLAG_HOSTNAME                0x100000
 
-#define FILTER_FLAG_EMAIL_UNICODE          0x100000
+#define FILTER_FLAG_EMAIL_UNICODE           0x100000
+
+/* FILTER_VALIDATE_STRING encoding options */
+#define FILTER_STRING_ENCODING_PASS              0
+#define FILTER_STRING_ENCODING_UTF8              1
+
+#define FILTER_FLAG_STRING_RAW              0x0001
+#define FILTER_FLAG_STRING_ALLOW_CNTRL      0x0002
```

Comment:
I think this should be named FILTER_FLAG_STRING_ALLOW_CTRL (without the 'n' in the end), as well as the userland constant as the constant name is already fairly long, and because CTRL is a more common abbreviation of control than CNTRL